client/dashboard/task-list/tasks/tax.js

```diff
@@ -227,15 +251,28 @@ class Tax extends Component {
 					'woocommerce-admin'
 				),
 				content: (
-					<Button
-						isPrimary
-						onClick={ () => {
-							recordEvent( 'tasklist_tax_config_rates' );
-						} }
-						href={ getAdminLink( 'admin.php?page=wc-settings&tab=tax&section=standard' ) }
-					>
-						{ __( 'Configure', 'woocommerce-admin' ) }
-					</Button>
+					<Fragment>
+						<Button isPrimary isBusy={ isPending } onClick={ this.configureTaxRates }>
+							{ __( 'Configure', 'woocommerce-admin' ) }
+						</Button>
+						<p>
+							{ interpolateComponents( {
+								mixedString: __(
+									'By clicking "Configure" you\'re enabling tax rates and calculations. More info {{link}}here{{/link}}.',
```

Comment:
Should we only show this if `woocommerce_calc_taxes` is no/not set?

Reply:
Definitely a good idea. Added in https://github.com/woocommerce/woocommerce-admin/pull/3168/commits/12cc0eb58a62cb1552c9bee75460332d24977a00